.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,107 @@
+title: "[Bug]: "
+labels: "bug"
+body:
+  - type: checkboxes
+    attributes:
+      label: Guidelines
+      description: Please ensure you've completed all of the following.
+      options:
+        - label: I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one i want to file, without success.
+          required: true
+        - label: I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).
+          required: true
+  - type: input
+    attributes:
+      label: FreeTube Nightly Build
+      description: What Nightly Build of FreeTube are you using? Go to the about page in the FreeTube application.  
+      placeholder: v0.14.0 build 1234
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: 'Installation Method (If applicable)'
+      options:
+        - .AppImage
+        - .deb
+        - .dmg
+        - .exe
+        - Flathub
+        - Portable
+        - .rpm
+        - .zip
+        - other
+    validations:
+      required: false
+  - type: dropdown
+    attributes: 
+      label: Primary API used
+      description: What is the primary API being used?
+      multiple: false
+      options:
+        - Local API
+        - Invidious API
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: What operating system are you using?
+      options:
+        - Windows
+        - macOS
+        - Ubuntu
+        - Other Linux
+        - Other (specify below)
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Operating System Version
+      description: What operating system version are you using? On Windows, click Start button > Settings > System > About. On macOS, click the Apple Menu > About This Mac. On Linux, use lsb_release or uname -a.
+      placeholder: "e.g. Windows 10 version 1909, macOS Catalina 10.15.7, or Ubuntu 20.04"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Last Known Working FreeTube Version
+      description: What is the last version of FreeTube this worked in, if applicable?
+      placeholder: v0.14.0
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Actual Behavior
+      description: A clear description of what actually happens.
+    validations:
+      required: true
+  - type: textarea
+    attributes: 
+      label: Steps To Reproduce
+      description: Please add all steps to reproduce the behavior
+      placeholder: |
+          1. Go to '...'
+          2. Click on '....'
+          3. Scroll down to '....'
+          4. See error
+    validations:
+      required: true
+  - type: input
```

Comment:
I think this should be `textarea` as there might be multiple URLs to videos

Reply:
This is removed therefore i will resolve this.